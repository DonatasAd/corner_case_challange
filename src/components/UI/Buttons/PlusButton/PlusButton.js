import React from 'react';
import BackArrow from '../../../../assets/plus-button.png';
import styles from './PlusButton.module.css';

export default function PlusButton(props) {
  return (
    <button {...props} className={styles.Button}>
      <img src={BackArrow} className={styles.Logo} alt='' />
    </button>
  );
}

PlusButton.defaultProps = {
  onClick: () => console.log('Plus button'),
};
