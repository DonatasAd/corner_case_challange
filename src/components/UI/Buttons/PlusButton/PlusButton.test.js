import React from 'react';
import { shallow } from 'enzyme';
import PlusButton from './PlusButton';

describe('PlusButton', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<PlusButton  />);
    expect(component).toMatchSnapshot();
  });
});