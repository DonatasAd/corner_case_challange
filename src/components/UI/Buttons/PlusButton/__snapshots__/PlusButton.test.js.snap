// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlusButton should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
